app/src/main/java/com/antosik/benchproject/app/more/view/More.kt

```diff
@@ -0,0 +1,20 @@
+package com.antosik.benchproject.app.more.view
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import com.antosik.benchproject.databinding.MoreFragmentBinding
+
+class More : Fragment() {
```

Comment:
Your other fragments are named `XXXFragment` but here you just named it `More`. Is it intended? I think it would be nice to stick to some convention – IMO `MoreFragment` would be better because `More` looks like some data class 😄.

Reply:
I looked at Favourite Movies
![image](https://user-images.githubusercontent.com/57491794/218399240-46acb60f-0414-4b4b-831f-375acd87f8bd.png)
 and I suggested it, so I will make an issue to rename Favourite Movies 😄